syntax = "proto2";

message StringRequest {
    required int64 string_id = 1;
}

message LocalizedString {
    required string value = 1;
}

service LocalizationHandler {
    rpc getString (StringRequest) returns (LocalizedString);
}